### ISSUE TITLE
[grid][bug] - Angular universal - group rows does not render

### ISSUE THREAD

**Issue by earlyster** [telerik/kendo-angular#968]
## I'm submitting a...
* Bug report 

## Current behavior

When I render grid in angular universal with grouped rows I receive no grid output and get error on console.

in function createDropCue there is a problem document is referenced causing issues with ng universal
```
var createDropCue = function (container) {
    var cue = document.createElement("div");
```

resulting html
```
<kendo-grid dir=ltr class="k-widget k-grid"></kendo-grid>
```

```
ERROR ReferenceError: document is not defined
    at createDropCue (dist/server.js:29505:15)
    at GroupConnectionService.register (dist/server.js:29535:24)
    at new GroupPanelComponent (dist/server.js:53776:14)
    at createClass (dist/server.js:11249:26)
    at createDirectiveInstance (dist/server.js:11067:37)
    at createViewNodes (dist/server.js:12515:49)
    at Object.createEmbeddedView (dist/server.js:12398:5)
    at TemplateRef_.createEmbeddedView (dist/server.js:10582:38)
    at ViewContainerRef_.createEmbeddedView (dist/server.js:10359:52)
    at NgIf._updateView (dist/server.js:17157:45)

```

## Expected behavior

Expected html with no server errors.

## Minimal reproduction of the problem with instructions

  1. http://plnkr.co/edit/CYOXlD8b5gT77yMrKRT1?p=preview
  2. Must run with angular universal starter project
   3. You will see error that looks like this in console running in nodejs server

## What is the motivation or use case for changing the behavior?

Rending grouped data on server to show grouped data within 300ms and improve user experience. 

## Environment

Package versions:
```
test
├── @angular/animations@4.4.4
├── @angular/common@4.4.4
├── @angular/compiler@4.4.4
├── @angular/compiler-cli@4.4.4
├── @angular/core@4.4.4
├── @angular/forms@4.4.4
├── @angular/http@4.4.4
├── @angular/platform-browser@4.4.4
├── @angular/platform-browser-dynamic@4.4.4
├── @angular/platform-server@4.4.4
├── @angular/router@4.4.4
├── @ngtools/webpack@1.7.2
├── @nguniversal/express-engine@1.0.0-beta.3
├── @progress/kendo-angular-dateinputs@1.2.0
├── @progress/kendo-angular-dropdowns@1.2.1
├── @progress/kendo-angular-excel-export@1.0.4
├── @progress/kendo-angular-grid@1.4.2
├── @progress/kendo-angular-inputs@1.3.1
├── @progress/kendo-angular-intl@1.2.2
├── @progress/kendo-angular-l10n@1.0.4
├── @progress/kendo-data-query@1.0.7
├── @progress/kendo-drawing@1.4.0
├── @types/express@4.0.37
├── @types/node@8.0.32
├── bootstrap@4.0.0-beta
├── css-loader@0.28.7
├── css-to-string-loader@0.1.3
├── express@4.16.1
├── express-minify-html@0.11.4
├── file-loader@0.11.2
├── html-webpack-plugin@2.30.1
├── UNMET PEER DEPENDENCY jquery@>=3.0.0
├── memory-cache@0.2.0
├── ngx-bootstrap@1.9.3
├── node-sass@4.5.3
├── nodemon@1.12.1
├── UNMET PEER DEPENDENCY popper.js@^1.11.0
├── raw-loader@0.5.1
├── rimraf@2.6.2
├── rxjs@5.4.3
├── sass-loader@6.0.6
├── script-ext-html-webpack-plugin@1.8.5
├── serialize-javascript@1.4.0
├── style-loader@0.18.2
├── typescript@2.5.3
├── url-loader@0.5.9
├── webpack@3.6.0
├── webpack-merge@4.1.0
├── webpack-node-externals@1.6.0
├── xhr2@0.1.4
└── zone.js@0.8.18
```

Browser:

Not applicable for browsers happens with server

System:
- TypeScript version: XX 
Version 2.2.2
- Node version: XX  
v6.9.5
- Platform:  
Mac

**Comment by rkonstantinov** [comment]
Hey @earlyster, thanks for the report. We were able to observe the issue in question and the fix will be included in the next version of the grid package.

**Comment by ipeshev** [comment]
Fixed in 1.5.0-dev.201710041221

**Comment by tsvetomir** [comment]
Fix published in v1.5.0
